src/store/actions/playerActions.js

```diff
@@ -25,3 +28,33 @@ export const getSongStreamLink = (path) => (dispatch, getState) => {
       throw new Error(err);
     });
 };
+
+export const playNextSong = () => (dispatch, getState) => {
+  const songIndex = getState().songsQueue.get('index');
+  const songsQueue = getState().songsQueue.get('queue');
+
+  const canPlayNextSong = songIndex + 1 <= songsQueue.size;
```

Comment:
You are duplicating the logic here and in every button's container. This means you can abstract this into a new logic layer: `selectors`.